rtl/include/eclhelper.hpp

```diff
@@ -363,6 +364,7 @@ struct RtlTypeInfo : public RtlITypeInfo
     inline bool isFixedSize() const { return (fieldType & RFTMunknownsize) == 0; }
     inline bool isLinkCounted() const { return (fieldType & RFTMlinkcounted) != 0; }
     inline bool isUnsigned() const { return (fieldType & RFTMunsigned) != 0; }
+    inline bool isValueNonScalarXpath() const { return (fieldType & RFTMhasnonscalarxpath) != 0; }
```

Comment:
really picky!: hasNonScalarXPath() would be clearer - I'm not sure why the word value is in here.  Will merge anyway.